src/cloudformation/newUserAndDbTemplate.yaml

```diff
@@ -0,0 +1,217 @@
+
+Description: 'IaSQL: user and databse creation'
+
+###
+### CloudFormation Interface Metadata
+###
+
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+    - Label:
+        default: IaSQL Connection Setup
+      Parameters:
+      - pIasqlDbName
+      - pAwsRegion
+      - pA0Token
+      - pIasqlUrl
+    ParameterLabels:
+      pIasqlDbName:
+        default: IaSQL Database Name
+      pAwsRegion:
+        default: AWS Region
+      pIasqlUrl:
+        default: IaSQL API URL
+      pA0Token:
+        default: IaSQL Bearer Token
+
+###
+#### Template input parameters
+####
+
+Parameters:
+  pIasqlDbName:
+    Type: String
+    Description: Do not update
```

Comment:
this means it can't be updated after it is created? Maybe "immutable after creation or something"

Reply:
oh this is the params we are passing from the ui. got it

Reply:
what I want to express is that users must not update the values creating the cloudformation template because those are the values from our UI and If they changed them here we loose track of them.